### PR TITLE
EVG-2062, EVG-2063, EVG-2064 implement display tasks on task page

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

**Comment by john-m-liu** [comment]
image

**Comment by tychoish** [comment]
thoughts: 
- maybe it makes sense to remove the logs area entirely, and put the "this is a display task" block elsewhere" 
- maybe it makes sense to put a panel with links to the executions where the logs would have been?
- most critically, I think it'll be useful to put links to the executions in the *test* block.

**Comment by john-m-liu** [comment]
image

**Comment by john-m-liu** [comment]
image

File: service/templates/task.html [hunk 1]
```diff
@@ -318,46 +324,50 @@ <h4 class="text-center text-muted">No test results to show</h4>
               <tr>
                 <th class="col-lg-6">Test</th>
                 <th class="col-lg-3">Time</th>
-                <th class="col-lg-3">Logs</th>
+                <th class="col-lg-3">Test Logs</th>
+                <th ng-if="task.display_only" class="col-lg-3">Other Logs</th>
```
Comment (tychoish) [inline service/templates/task.html]: I'd call this "Execution Task"

Comment (john-m-liu) [reply service/templates/task.html]: I changed it to have both to make it more obvious that logs can be found there

**Pull request closed without merging**